cmd/serv.go

```diff
@@ -112,8 +111,7 @@ func runServ(c *cli.Context) error {
 	}
 
 	if len(c.Args()) < 1 {
-		cli.ShowSubcommandHelp(c)
-		return nil
+		return cli.ShowSubcommandHelp(c)
```

Comment:
Sorry, missed this. This is different from previous on logic.

Reply:
So the error from `cli.ShowSubcommandHelp(c)` should just be logged and then be ignored?

Reply:
I think yes

Reply:
Fixed.